docs/fsharp-cheatsheet.md

```diff
@@ -134,15 +132,15 @@ Pipe operator `|>` is used to chain functions and arguments together. Double-bac
 This operator can assist the F# type checker by providing type information before use:
 
     let sumOfLengths (xs : string []) =
-        xs 
+        xs
         |> Array.map (fun s -> s.Length)
         |> Array.sum
 
 Composition operator `>>` is used to compose functions:
 
     let squareNegateThenPrint' =
```

Comment:
this was a ref to L125 a) illustrating `'` b) illustrating the correspondence between `>>` and parenthesized nested calls

Reply:
I cleared all trailing white space. Didn't realize how man lines were changed until after. :-/